bot/moderation/moderation.py

```diff
@@ -771,7 +771,7 @@ async def modmail(self, ctx: commands.Context):
         Args:
             ctx (discord.ext.commands.Context): The context in which the command was called.
         """
-        if not self._db_connector.is_botonly(ctx.channel.id):
+        if ctx.channel.type.name not in ["private", "group"] and not self._db_connector.is_botonly(ctx.channel.id):
```

Comment:
what datatype is `ctx.channel.type`? Is it something like enums? If so it would be better to directly go on enum types, bc their string values might change.